packages/widgets-manager/src/manager.spec.ts

```diff
@@ -148,7 +193,49 @@ describe('setupManager', () => {
       expect(mountedApp).toBeDefined();
 
       expect(mountedApp!.initialData).toMatchObject(mockInitialData);
-    })
+    });
+
+    describe('When mounting apps with mode popup', () => {
+      let manager: AppManager;
+      const mockWindowOpen = jest.fn();
+      const mockInitialData = {
+        roomId: 'ABC-123',
+      };
+      beforeEach(() => {
+        manager = setupManager([mockPopupApp], mockFetchAppData);
+        window.open = mockWindowOpen;
+      });
+
+      it('Mounts the app', async () => {
+        await manager.mountApp(4, mockInitialData);
```

Comment:
It would be more descriptive if `mountApp` would be called with the app-name instead of the app-id.

Reply:
https://github.com/lets-talk/chat-utils/issues/57